rhc-ose-ansible/integration.yaml

```diff
@@ -0,0 +1,23 @@
+---
+- hosts: localhost
+  vars_prompt:
```

Comment:
@JayKayy the point is not to avoid having an inventory file. The point is that we will be feeding in an rhc-ose specific inventory file for our stuff... and as part of the provisioning process, we want to "produce" an inventory file that looks like what you'd expect for an openshift-ansible inventory. This is for the purposes of debugging/troubleshooting openshift-ansible stuff vs rhc-ose stuff.